portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -55,21 +70,43 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String comment = getParameter(request, "text-input", "");
+    String comment = getStringParameter(request, "text-input", "");
```

Comment:
It feels like `"text-input"` should be a `static final` at the top of your class like other request parameters.